scripts/toolboxSrc/pushToCDN.ts

```diff
@@ -11,7 +11,7 @@ const pushToCDN = async () => {
   const context = (require as any).context(
     '../../static/images/illustrations',
     false,
-    /Template.png$/
+    /\/action.png$|\/teamPrompt.png$|Template.png$/
```

Comment:
we don't suffix these fixed activities with `Template`, which is fine, but it wasn't grabbing them to pushToCDN